From ea69af32f973e99819ddabbbf687aeca17aa236b Mon Sep 17 00:00:00 2001
From: Nicolin Chen <B42378@freescale.com>
Date: Sat, 8 Jun 2013 17:17:33 +0800
Subject: [PATCH 398/442] ENGR00266287 mxc: tuner: search si4763 in script

Previously we used a fixed number, that's no good, so add search code.

Signed-off-by: Nicolin Chen <B42378@freescale.com>
---
 test/mxc_tuner_test/mxc_tuner_test.sh | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/test/mxc_tuner_test/mxc_tuner_test.sh b/test/mxc_tuner_test/mxc_tuner_test.sh
index a6a4c9f..9dd11ae 100644
--- a/test/mxc_tuner_test/mxc_tuner_test.sh
+++ b/test/mxc_tuner_test/mxc_tuner_test.sh
@@ -1,8 +1,10 @@
 #!/bin/sh
 
-arecord -D hw:2,0 -f dat | aplay -D hw:0,0 -f dat &
+num=`arecord -l |grep -i "si4763" |awk '{ print $2 }'|sed 's/://'`
+
+arecord -D hw:$num,0 -f dat | aplay -D hw:0,0 -f dat &
 ./mxc_tuner_test.out
-pid=$(ps | grep "arecord -D hw:2,0 -f dat" | grep -v ' grep ' | awk '{print $1}')
+pid=$(ps | grep "arecord -D hw:$num,0 -f dat" | grep -v ' grep ' | awk '{print $1}')
 echo "killing pid = $pid"
 kill $pid
 
-- 
1.8.0

